src/clj/game/core/actions.clj

```diff
@@ -163,7 +163,7 @@
   (let [card (get-card state card)
         r (get-in @state [side :selected 0 :req])
         cid (get-in @state [side :selected 0 :not-self])]
-    (when (and (not= (:cid card) cid) (or (not r) (r card))
+    (when (and (not= (:cid card) cid) (or (not r) (r card)))
```

Comment:
How has this not caused heaps of issues??
Ah, probably lazy-evaluation. At least it's clearer this way

Reply:
haha Yeah, I have no idea how this hasn't messed everything up.